draft-ietf-core-coap-tcp-tls.md

```diff
@@ -737,36 +737,45 @@ Capabilities and Settings messages are indicated by the 7.01 code (CSM).
 
 ### Server-Name Setting Option
 
-| Number | Applies to | Name                | Format     | Length      | Base Value  |
-|:-------+:-----------+:--------------------+-----------:+------------:+------------:+
-|      1 | CSM        | Server-Name         | string     | 1-255       | (see below) |
+|No|C|R| Applies to | Name               | Format | Length | Base Value  |

```

Comment:
Some people will read "No" as "no" (as opposed to Yes).  Maybe "#" is a better short title?